testsuites/CBLTester/system_tests/test_system_concurrent_testing.py

```diff
@@ -0,0 +1,384 @@
+import pytest
+import time
+import random
+from sys import maxsize
+import concurrent.futures
+
+from keywords.MobileRestClient import MobileRestClient
+from CBLClient.Replication import Replication
+from CBLClient.Authenticator import Authenticator
+from keywords.utils import log_info
+from libraries.testkit.cluster import Cluster
+from libraries.data.doc_generators import simple, four_k, simple_user, complex_doc
+from datetime import datetime, timedelta
+from CBLClient.Utils import Utils
+
+
+@pytest.mark.listener
+@pytest.mark.replication
+def test_system(params_from_base_suite_setup):
+    sync_gateway_version = params_from_base_suite_setup["sync_gateway_version"]
+    if sync_gateway_version < "2.0.0":
```

Comment:
Like Functional test summary , can you please write the steps? Also match it with the comments. For future quick debug.

Reply:
added summary and steps